Assets/MixedRealityToolkit/Utilities/Editor/PackageManifest/PackageManifestUpdater.cs

```diff
@@ -102,11 +187,12 @@ internal static void EnsureMSBuildForUnity()
             int dependenciesStartIndex = -1;
             int scopedRegistriesStartIndex = -1;
             int scopedRegistriesEndIndex = -1;
+            int packageLine = -1;
 
             // Attempt to find the MSBuild for Unity package entry in the dependencies collection
             // This loop also identifies the dependecies collection line and the start / end of a
             // pre-existing scoped registries collections
-            bool needToAddPackage = true;
+            bool addPackage = true;
```

Comment:
Nit: I feel like the old naming here was actually more appropriate especially given the comments (i.e. if no package was found add it the dependencies collection).

Though I think in that case just changing the comment would also make things feel a little better.